lib/interfaces/plugins.js

```diff
@@ -262,13 +289,23 @@ module.exports = function (app) {
         } else {
           stopPlugin(plugin)
           plugin.start(newConfiguration, restart)
+          setPluginStartedMessage(plugin)
         }
       })
     }
     if (options && options.enabled) {
       debug('Starting plugin %s from %s', pluginName, location)
-      plugin.start(getPluginOptions(plugin.id).configuration, restart)
-      debug('Started plugin ' + pluginName)
+      try {
+        plugin.start(getPluginOptions(plugin.id).configuration, restart)
```

Comment:
We are calling plugin.start, debug and setPluginStartedMessage in several places. Sounds like we need `doStartPlugin(...)` that does these consistently.